packages/electron/src/client/main.js

```diff
@@ -63,6 +63,9 @@ module.exports = (opts) => {
   bugsnag.markLaunchComplete = markLaunchComplete
 
   bugsnag._logger.debug('Loaded! In main process.')
+  if (bugsnag._config.enabledBreadcrumbTypes && bugsnag._config.enabledBreadcrumbTypes.includes('navigation')) {
```

Comment:
Is there value in a loaded breadcrumb in renderers too?

Mostly thinking of cases where a renderer is started while using an app, which could be a long time after the app is opened

Reply:
It was agreed with product that it would be just in the main process.

There is the breadcrumb on the `browser-window-created` event which would give you the same info though?